src/pages/CommandView/form-data/get-validator/get-integer-validators.ts

```diff
@@ -0,0 +1,98 @@
+import { FormValidation } from '@rjsf/core'
```

Comment:
This provides a validator for `Integer` parameters, since python has a distinction between ints and floats, but javascript only has `number`.